core/request_handler.go

```diff
@@ -262,11 +262,13 @@ func sendReply(ch *Channel, reply *vppReply) {
 		}).Warn("Reply channel full, dropping reply.")
 		return
 	}
+	replyTimeoutTimer := time.NewTimer(ch.receiveReplyTimeout)
+	defer replyTimeoutTimer.Stop()
 	select {
 	case ch.replyChan <- reply:
 		return // reply sent ok
-	case <-time.After(ch.receiveReplyTimeout):
-		// receiver still not ready
+	case <-replyTimeoutTimer.C:
+		// receiver still isn't ready
```

Comment:
I understand the idea but I don't understand the issue.
time.After creates a temporary leak until the timer expires. After the expiration, it is supposed to be garbage collected. By default the timeout is 100ms. So I am not sure how you can fire enough API calls in such a little time window to create a significant memory impact.

Reply:
@hardeker hello Arthur, two reasons:
1. the reply timeout is indeed 100ms by default, but is configurable via `ReplyChannelTimeout` so the garbage collection can be easily and unknowingly by a user "postponed".
2. It depends on what we consider a "significant memory impact". I understand the impact is temporary, but memory spikes are noticeable as the timer is responsible for >30% of total GoVPP memory consumption. Or even much more, see this example: 
![VirtualBox_CodeFoundry_16_01_2024_10_14_36](https://github.com/FDio/govpp/assets/15376606/d7e67621-f82c-481f-9cf1-6b534bad52c9)